indexer/src/main/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtEntryConverter.java

```diff
@@ -150,22 +149,31 @@ private void convertEvidenceSources(UniProtKBEntry uniProtkbEntry, UniProtDocume
                 evidences.stream()
                         .map(Evidence::getEvidenceCrossReference)
                         .filter(Objects::nonNull)
-                        .map(CrossReference::getDatabase)
                         .filter(
-                                val ->
-                                        (val != null)
-                                                && val.getEvidenceDatabaseDetail().getCategory()
+                                xref ->
```

Comment:
can xref ever be null? it looks as though we had a (val != null) in the previous version

Reply:
In think it did not show the code block, but we have a filter above

                        .map(Evidence::getEvidenceCrossReference)
                        .filter(Objects::nonNull)
                        .filter(
                                xref -> ..  I removed the null check here...